index.js

```diff
@@ -1057,12 +1057,22 @@
   //. of the first occurrence of the value in the list, if applicable;
   //. Nothing otherwise.
   //.
+  //. Dispatches to its second argument's `indexOf` method if present.
+  //. As a result, `String -> String -> Maybe Number` is an alternative
+  //. type signature.
+  //.
```

Comment:
@buzzdecafe, @CrossEye, @paldepind, @raine: What do you think of this wording? Should we include a similar paragraph in the docstring of each Ramda function which dispatches? (The second sentence would not be required in all cases.)